interop-tests/chromium-ping-version.json

```
@@ -0,0 +1,7 @@
+{
+  "id": "chromium-rust-libp2p-head",
+  "containerImageID": "chromium-rust-libp2p-head",
+  "transports": ["webtransport"],
```

```suggestion
  "transports": [{ name: "webtransport", onlyDial: true }],
```

See https://github.com/libp2p/test-plans/blob/01436349273084ef02819eb8525af6ceb18b68ca/multidim-interop/versions.ts#L93.

---

done